container/README.md

```diff
@@ -0,0 +1,110 @@
+# watson-conversation-slots-intro-kube
+
+This directory allows you to deploy the watson-conversation-slots-intro application into a container running on Bluemix. 
+
+# Steps
+
+## Create the Kubernetes cluster
+
+* Follow the instructions to [Create a Kubernetes Cluster](https://github.com/IBM/container-journey-template). When you get to step 2 name your cluster Watson. 
+```
+$ bx cs cluster-create --name Watson
+```
+* If you choose a different name or already have a cluster, please substitute that name for 'Watson' in the instructions that follow.
+
+## Create the Watson Conversation Service and Bind to your Cluster
+
+Either follow the instructions to [Create a Conversation Service](https://console.ng.bluemix.net/catalog/services/conversation) or Perform the following from the Bluemix CLI:
+
+* Create the Watson Conversation service.
+
+```
+bx service create conversation free 
+	conversation-service-watson-pizzeria
+```
+
+
+* Verify that the service instance is created.
+
+```
+bx service list
+```
+
+* Obtain the ID of your cluster.
+
+```
+bx cs clusters
+```
+
+* Bind the service instance to your cluster.
+
+```
+bx cs cluster-service-bind <cluster-ID> default conversation-service-watson-pizzeria
+```
+
+## Load the Watson Conversation
+
+* Follow the instructions to [Load the Watson Conversation](https://github.com/IBM/watson-conversation-slots-intro#4-configure-watson-conversation)
+
+## Create a Kubernetes Configuration Map with the Workspace ID
+
+* Create a Kubernetes Configuration Map
+
+```kubectl create configmap watson-pizzeria-config --from-literal=workspace_id=<WorkSpace_ID>```
```

Comment:
code blocks should be either single backtick on same line,  or triple backtick newline code, triple backtick.